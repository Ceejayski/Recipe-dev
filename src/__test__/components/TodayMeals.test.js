import { Provider } from 'react-redux';
import { create } from 'react-test-renderer';
import TodayMeal from '../../components/todayMeal';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <TodayMeal />
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
