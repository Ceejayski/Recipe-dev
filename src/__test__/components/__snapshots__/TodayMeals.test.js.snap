// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
Array [
  <h3
    className="mt-5 text-center"
  >
    Recipes of the Day
  </h3>,
  <div
    className="row"
  />,
]
`;
